app-emulation/playonlinux/playonlinux-4.4.ebuild

```diff
@@ -0,0 +1,118 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7,8,9} )
+
+inherit desktop eutils python-single-r1 xdg
+
+MY_PN="PlayOnLinux"
+
+DESCRIPTION="Set of scripts to easily install and use Windows games and software"
+HOMEPAGE="https://playonlinux.com/"
+SRC_URI="https://www.playonlinux.com/script_files/${MY_PN}/${PV}/${MY_PN}_${PV}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="winbind"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+DEPEND=""
+RDEPEND="${PYTHON_DEPS}
+	app-arch/cabextract
+	app-arch/p7zip
+	app-arch/unzip
+	app-crypt/gnupg
+	app-misc/jq
+	$(python_gen_cond_dep '
+		dev-python/natsort[${PYTHON_USEDEP}]
+		dev-python/wxpython:4.0[${PYTHON_USEDEP}]
+	')
+	media-gfx/icoutils
+	|| ( net-analyzer/netcat net-analyzer/openbsd-netcat )
+	net-misc/wget
+	virtual/imagemagick-tools
+	virtual/wine
+	winbind? ( net-fs/samba[winbind] )
+	x11-apps/mesa-progs
+	x11-terms/xterm
+"
+
+S="${WORKDIR}/${PN}"
+
+# TODO:
+# Having a real install script
+# It will let using LANGUAGES easily
+# How to deal with Microsoft Fonts installation asked every time ?
+# How to deal with wine version installed ? (have a better mgmt of system one)
+# Look at debian pkg: https://packages.debian.org/sid/playonlinux
+
+PATCHES=(
+	"${FILESDIR}/${PN}-4.2.4-pol-bash.patch"
+	"${FILESDIR}/${PN}-4.2.4-binary-plugin.patch"
+	"${FILESDIR}/${PN}-4.2.6-stop-update-warning.patch"
+	"${FILESDIR}/${PN}-4.4-fix-broken-version-check.patch"
+	"${FILESDIR}/${PN}-4.4-wxpython4-fix.patch"
+)
+
+src_prepare() {
+	default
+
+	python_fix_shebang .
+
+	# remove playonmac
+	rm etc/{playonmac.icns,terminal.applescript} || die
+
+	# remove desktop integration
+	rm etc/{PlayOnLinux.desktop,PlayOnLinux.directory,playonlinux-Programs.menu} || die
+
+	# fix Makefile to avoid eclass error
+	sed -i '/^PYTHON =/s/python2/python/' Makefile || die
+
+	# fix find_python bash script to find EPYTHON
+	sed -i '/[ \t]*next_python "python3"/s/^/next_python "'${EPYTHON}'"\n/' \
+		bash/find_python || die
```

Comment:
When you use a direct command like cp, mv, sed, rm please add a short description on what's failed. For eg `die "Failed to remove playonmac's files"`